libs/types/src/auth/auth.ts

```diff
@@ -17,6 +17,16 @@ export interface ElectionKey {
   date: DateWithoutTime;
 }
 
+/**
+ * Create an {@link ElectionKey} from an {@link Election}.
+ */
+export function electionAuthKey(election: Election): ElectionKey {
```

Comment:
Extreme nit that you're free to ignore: Could call this `constructElectionKey` or `electionKeyFromElection` instead of `electionAuthKey` since "election auth key" sounds like something distinct from an "election key"

Reply:
Reasonable feedback! 6eb372e88